src/client/app/utils/notifications.js

```diff
@@ -0,0 +1,24 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import store from '../index';
+import { showNotification as showNotificationAction } from '../actions/notifications';
+
+export function showSuccessNotification(message, position = 'tr', autoDismiss = 3) {
```

Comment:
To forestall future comments on this technique (exporting store and dispatching on it), this is, as far as I can tell, the best way to create a global dispatcher without having to modify every Container that could possibly have errors.